src/app/example-module/example-module-routing.module.ts

```diff
@@ -2,20 +2,27 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
 import { CardModule } from 'primeng/card';
+import { MenubarModule } from 'primeng/menubar';
+import { ButtonModule } from 'primeng/button';
+
 import { ExampleComponentComponent } from './example-component/example-component.component';
+import { MenubarComponent } from '../common/menubar/menubar.component';
 
 const routes: Routes = [
   { path: '', component: ExampleComponentComponent, pathMatch: 'full' },
   { path: 'example', component: ExampleComponentComponent },
+  { path: 'bar', component: MenubarComponent }
```

Comment:
eee?

Reply:
rozumiem, że to jakieś stare i niezmienione? xd

Reply:
To prawdopodobnie wziąło się z jakiś rebasów, nic nie zmieniałem w tym miejscu